src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/DataStreams.cs

```diff
@@ -126,9 +126,7 @@ private ArrayList SaveSubStreams(UIElement element)
                         {
                             // Convert the value of the DP into a byte array
                             MemoryStream byteStream = new MemoryStream();
-                            #pragma warning disable SYSLIB0011 // BinaryFormatter is obsolete 
-                            this.Formatter.Serialize(byteStream, currentValue);
-                            #pragma warning restore SYSLIB0011 // BinaryFormatter is obsolete 
+                            BinaryFormatWriter.TryWriteFrameworkObject(byteStream,currentValue);
```

Comment:
Do you know that this works in all cases? Why are you not checking for success here?

Reply:
I have added a flag ```success```, on getting false it will use BinaryFormatter.